backend/src/test/repository/UserRepository.test.ts

```diff
@@ -0,0 +1,25 @@
+import { getConnection, getCustomRepository } from 'typeorm';
+import { User } from '../../main/entity/User';
+import { users_sample } from '../../main/resources/Data';
+import { UserRepository } from '../../main/repository/UserRepository';
+import { BaseRepositoryTest } from './BaseRepositoryTest';
+
+describe('Unit tests for UserRepository', function () {
+  beforeEach(() => {
+    return BaseRepositoryTest.establishTestConnection();
```

Comment:
This will establish a connection with our `foodies_test` database each time, so it won't interfere with `foodies` database. 

Originally, I wanted to create a test hierarchy with the base class being `BaseRepositoryTest`. But this was a quicker solution with a static method.